src/config/app.config.ts

```diff
@@ -115,4 +115,5 @@ export default registerAs('app', () => ({
     'EASEY_AUTH_API_SIGN_FILES_INDIVIDUALLY',
     false,
   ),
+  maintenanceBypassUsers:getConfigValue('EASEY_MAINTENANCE_BYPASS_USERS',[])
```

Comment:
This will need to be passed in as stringified JSON (since `getConfigValue` does not parse the environment value), so the default value should reflect that. It will also need to parsed, either here or where it is used. For example, in the current form, `'["MDIEBOLD_DP"]'.includes('M')` would return true, which is not what we want.
```suggestion
  maintenanceBypassUsers: JSON.parse(getConfigValue('EASEY_MAINTENANCE_BYPASS_USERS', '[]')),
```